boards.txt

```diff
@@ -398,6 +398,94 @@ metro52840.menu.debug.l3.build.sysview_flags=-DCFG_SYSVIEW=1
 #
 # -------------------------------------------------------
 
+# ----------------------------------
+# Opossum Pouch
+# ----------------------------------
+op1.name=Opossum Pouch nRF52833 (op1)
+op1.bootloader.tool=bootburn
+
+# Upload
+op1.upload.tool=nrfutil
+op1.upload.protocol=nrfutil
+op1.upload.use_1200bps_touch=true
+op1.upload.wait_for_upload_port=true
+op1.upload.maximum_size=290816
+op1.upload.maximum_data_size=52224
+
+# Build
+op1.build.mcu=cortex-m4
+op1.build.f_cpu=64000000
+op1.build.board=NRF52833_op1
+op1.build.core=nRF5
+op1.build.variant=opossum_pouch_nrf52833
+op1.build.usb_manufacturer="Nordic"
+op1.build.usb_product="nRF52833"
+op1.build.extra_flags=-DNRF52833_XXAA {build.flags.usb}
+op1.build.ldscript=nrf52833_s140_v6.ld
+op1.build.vid=0x239A
+op1.build.pid=0x8029
+
+# SofDevice Menu
+op1.menu.softdevice.s140v6=0.5.0 SoftDevice s140 6.1.1
+op1.menu.softdevice.s140v6.build.sd_name=s140
+op1.menu.softdevice.s140v6.build.sd_version=6.1.1
+op1.menu.softdevice.s140v6.build.sd_fwid=0x00B6
+
+# Debug Menu
+op1.menu.debug.l0=Level 0 (Release)
+op1.menu.debug.l0.build.debug_flags=-DCFG_DEBUG=0
+op1.menu.debug.l1=Level 1 (Error Message)
+op1.menu.debug.l1.build.debug_flags=-DCFG_DEBUG=1
+op1.menu.debug.l2=Level 2 (Full Debug)
+op1.menu.debug.l2.build.debug_flags=-DCFG_DEBUG=2
+op1.menu.debug.l3=Level 3 (Segger SystemView)
+op1.menu.debug.l3.build.debug_flags=-DCFG_DEBUG=3
+op1.menu.debug.l3.build.sysview_flags=-DCFG_SYSVIEW=1
+
+
+# ----------------------------------
+# Generic NRF52833
```

Comment:
generic board is not accepted, this should be changed to pca10100 if that is what you mean.

Reply:
Yes, that's the intent. I was reworking another commit and missed the documentation name change.